### ISSUE TITLE
Hi, How do you monitor the training process of the network?

### ISSUE THREAD

**Issue by pengfeidip** [1991viet/Yolo-pytorch#12]
Hi
in #6 ,  I know that you have not implemented mAP calculations, so how do you monitor your training process? or just by loss curve ？

**Comment by 1991viet** [comment]
Hi,
Yes, by the loss curve. Of course at the end, we need to evaluate the trained model based on mAP (which I have not implemented yet). However, during the training, the loss is used to measure the training progress, and if we want to apply stuff like early stopping, the loss is also used. We cant say (although it is true) sth like "Our model at the end of epoch 10 outperforms our model at the end of epoch 1, because the former's mAP is higher than the latter's one

**Comment by pengfeidip** [comment]
Thank you .... Because my friends told me that I should monitor **loss and mAP** during training process...

Can you help me to analysis following loss curve? I try to train a YOLO-tiny_v1(9 layers)from scratch on VOC 2007. training set is `voc 2007 train`, val set is `voc 2007 val`. I think the loss curve is strange...
![image](https://user-images.githubusercontent.com/23066797/49694702-bf394580-fbc9-11e8-9fae-7d1a96188565.png)  
(y is loss, red is `training loss`, blue is `val loss`, x is num of epoch)
